### PR TITLE
Typescript init

### PR DESCRIPTION
**Pull request by jchavez443**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,46 +1,52 @@
 {
-  "name": "@bandwidth/voice",
-  "version": "1.5.0",
-  "description": "Bandwidth's HTTP voice platform",
-  "main": "./lib/index.js",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/bandwidth/node-voice.git"
+  "version": "1.0.0",
```
Comment (jmulford-bw) [inline package.json:2]: Should probably make this `2.0.0`?

**Review by jmulford-bw — CHANGES_REQUESTED** [review]
1 small change. I'm assuming we've used this against our api, right?

File: package.json (hunk 1 above)
Comment (jchavez443) [reply package.json:2]: Oh yea for sure

Comment (jchavez443) [reply package.json:2]: We also probably will be adding this file to the ignore list.  So we can go through it and modify things.  
Like getting ride of some of the APIMatic references.  But lets do that on a cleaner PR.

**Review by jmulford-bw — APPROVED** [review]

**Review by ajrice6713 — APPROVED** [review]

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name: Deploy to NPM
+
+on:
+  push:
+    tags:
+      - 'v*'
```
Comment (hamermike) [inline .github/workflows/deploy.yml:6]:
```
release:
  types:
    - published
```

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+jobs:
+  deploy:
+    name: Push the Node SDK npm
+    runs-on: ubuntu-latest
```
Comment (hamermike) [inline .github/workflows/deploy.yml:11]: ```if: ${{ !github.event.release.prerelease && github.event.release.target_commitish == 'main' }}```

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (hamermike) [reply .github/workflows/deploy.yml:6]: Small change to deploy on a published release. Below comment restricts it further to the main branch.

**Review by hamermike — CHANGES_REQUESTED** [review]

**Review by hamermike — APPROVED** [review]

**Pull request merged**